.github/workflows/pr-description-check.yml

```
@@ -25,25 +25,24 @@ jobs:
             ## Test plan
             ## Documentation update
         run: |
-          # Read required sections from env var, one per line
           readarray -t required_sections <<< "$REQUIRED_SECTIONS"
-          pr_body="${{ github.event.pull_request.body }}"
+          pr_body_base64=$(echo -n "${{ github.event.pull_request.body }}" | base64 -w0)
           missing_sections=()
 
           for section in "${required_sections[@]}"; do
-            if [[ -z $(echo "$pr_body" | grep -i "$section") ]]; then
+            if ! echo "$pr_body_base64" | base64 --decode | grep -q -i "$section"; then
```

if the issue was substitution, base64-encoding shouldn't be necessary, right?

```suggestion
          pr_body="${{ github.event.pull_request.body }}"
          missing_sections=()

          for section in "${required_sections[@]}"; do
            if ! echo "$pr_body" | grep -q -i "$section"; then
```

---

Tested this in private repository check [here](https://github.com/frdomovic/stellar-json-rpc/actions/runs/14197121277/job/39774810647?pr=22). It breaks again because it tries to execute the command written in description

---

weird